### PR TITLE
Completed the EJS files. When doing the code review you can focus on …

### PR DESCRIPTION
**Pull request by latoyazamill**
…the EJS files only. The HTML files is what we worked with first to get the base for our EJS templates.

### CODE REVIEW

File: app/views/pages/albums.ejs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<%- include ('../shared/header', {'pageTitle' : 'Albums'}); %>
+  <h1 class="white-text">Albums</h1>
+  <div class="divider"></div>
+  <div class="row"></div>
+  <% albums.forEach(function(albums){%}>
```
Comment (lizzkats) [inline app/views/pages/albums.ejs]: I don't think you meant to have that closing curly brace here

**Review by lizzkats — COMMENTED** [review]
Woo! Good job 👍

File: app/model/artist.js [hunk 2]
```diff
@@ -14,7 +14,6 @@ Artist.prototype.getArtist = function(id)
 
 Artist.prototype.loadAll = function(callback)
 {
-  console.log("Reading files...")
   fs.readFile('db/artists.json', 'utf8', function (err, data) {
     if (err) {
       return console.log(err);
```
Comment (lizzkats) [inline app/model/artist.js:19]: Another way to do this is console.error which makes your errors easier to read :)

File: app/views/pages/albums.ejs (hunk 1 above)
Comment (latoyazamill) [reply app/views/pages/albums.ejs]: Thanks! Complete :)

**Pull request merged**